src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -4273,6 +4379,64 @@ components:
             validators:
               $ref: "#/components/schemas/ValidatorUptimeCollection"
 
+    #
+    # /wallet/* Request & Response Types
+    #
+
+    WalletTbdRequest:
+      allOf:
+        - $ref: "#/components/schemas/AtLedgerStateMixin" # TODO do we even want to support that? if yes, it could technically fall under regular /state/* endpoints
```

Comment:
Yeah I think we can support this relatively easily if we store the fully history. I think we may as well just store the full history.

But I don't think it's necessary a `/state/*` endpoint conceptually.

Reply:
I'll leave it to future us to decide if we want to support that or not.

Reply:
We should decide now I think to make implementation run smoothly. Let's say "yes" unless you object strongly.